### PR TITLE
Feat/int 686 allow to configure the include and

### PR DESCRIPTION
**Pull request by henoktsegayetoptal**

### CODE REVIEW

File: src/components/engineService.ts [hunk 1]
```diff
@@ -177,27 +177,29 @@ export class EngineService {
...
 
 			if (message.command.engine === 'node' && 'uri' in message.command) {
 				const commandUri = message.command.uri;
 
-				['js', 'jsx', 'ts', 'tsx'].forEach((extension) => {
-					const { fsPath } = Uri.joinPath(
-						commandUri,
-						`**/*.${extension}`,
-					);
+				includePatterns.forEach((extension) => {
```
Comment (grzpab) [inline src/components/engineService.ts:189]: `extension` => `includePattern`

File: src/components/engineService.ts [hunk 2]
```diff
@@ -177,27 +177,29 @@ export class EngineService {
...
+					const { fsPath } = Uri.joinPath(commandUri, extension);
 
 					const path = singleQuotify(fsPath);
 
 					args.push('-p', path);
 				});
+				excludePatterns.forEach((extension) => {
```
Comment (grzpab) [inline src/components/engineService.ts:196]: `excludePatterns` => `excludePattern`

File: src/components/engineService.ts [hunk 3]
```diff
@@ -206,6 +208,7 @@ export class EngineService {
 				);
 
 				args.push('-l', String(fileLimit));
+				console.log('args', args);
```
Comment (grzpab) [inline src/components/engineService.ts:211]: can you remove it?

Comment (henoktsegayetoptal) [reply src/components/engineService.ts:211]: wow I keep forgetting this

File: src/components/engineService.ts (hunk 2 above)
Comment (henoktsegayetoptal) [reply src/components/engineService.ts:196]: it is an array of excludePattern

**Pull request merged**